### PR TITLE
resolves 7: add project report to antora pages

### LINKED ISSUE #7: Add Project Report to Antora pages

### PR DESCRIPTION
**Pull request by prudhomm**
- closes #7

### CODE REVIEW

File: docs/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+.xref:index.adoc[Introduction]
 * Solar Shading environment
 ** xref:cmake.adoc[cmake environment]
 ** xref:antora.adoc[antora environment]
 ** xref:vscode.adoc[vscode integration]
 ** xref:githubactions.adoc[Github Actions]
 ** xref:rename.adoc[Renaming the project]
 ** xref:jupyter.adoc[Jupyter Notebook]
-
+* xref:rapport.adoc[Rapport]
```
Comment (lberti) [inline docs/modules/ROOT/nav.adoc:9]: Please put [Report Ariel De Vora]

File: docs/modules/ROOT/pages/antora.adoc [hunk 2]
```diff
@@ -9,7 +9,7 @@ It is part of the documentation of docs.feelpp.org[{feelpp} docs] website.
 ----
  $ npm install # <1>
  $ npm run antora # <2>
- $ npm run serve # <3>
+ $ npm run start # <3>
```
Comment (lberti) [inline docs/modules/ROOT/pages/antora.adoc:12]: `npm run serve` is not working?

**Review by lberti — CHANGES_REQUESTED** [review]
Please ensure that all pictures cite the references where they were taken from

File: docs/modules/ROOT/pages/antora.adoc (hunk 2 above)
Comment (EarltShirt) [reply docs/modules/ROOT/pages/antora.adoc:12]: the serve script to run `npm run serve` seems to be missing, but `npm run start` seems to do run the server properly

**Review by lberti — APPROVED** [review]

**Pull request merged**